### PR TITLE
Create Election Guidelines

### PR DESCRIPTION
**Pull request by DavidPHirsch**
adding election process

### CODE REVIEW

**Comment by DavidPHirsch** [comment]
@beeme1mr can you also share this in the GC slack channel?

File: Elections/2023/Election-Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+We highly encourage participation in this election cycle to ensure that the community is well-represented by the Governance Committee.
+
+# TL;DR
+
+* If you've been nominated or are willing to nominate yourself: express your interest in [this issue](https://github.com/open-feature/community/issues/262)
+* Vote between 17 November 2023 00:00 UTC and 17 December 2023 23:59 UTC via the [voting link](https://vote.heliosvoting.org/helios/e/OpenFeature2023)
+
+# Vacancies
+TThis election should fill seven seats:
```
Comment (AlexsJones) [inline Elections/2023/Election-Guidelines.md:24]:
```suggestion
This election should fill seven seats:
```

File: Elections/2023/Election-Guidelines.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* GitHub alias
+* Company affiliation (if applicable)
+* Short bio or reasoning to join the Governance Committee (no more than a short paragraph)
+* _Optional_: photo/picture of a nominee
+
+The Pull Request will not be merged until the candidate has confirmed their desire to be nominated (if not self-nominating) and ratified via PR comments.
+
+# Voter Eligibility
+
+All [members of the OpenFeature Organization Members, Approvers, Maintainers will automatically be eligible to vote.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:53]: Was this meant to be a link?

File: Elections/2023/Election-Guidelines.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+This election should fill seven seats:
+- fours seats from the existing governance committee
+- three seats from new candidates
+To encourage diversity there will be a maximum of one-third representation on the Governance Committee from any one company at any time. 
+If the outcomes of an election result in greater than 1/3 representation (or maximum of two, whichever is greater), the lowest vote getters from any particular company will be removed until representation on the committee is equal or less than one-third.
+
+
+# Voting process
+
+Anyone can track the 2023 election process via [this GitHub issue](https://github.com/open-feature/community/issues/262). We will ensure that all documents and assets related to the 2023 election process are public.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:33]:
```suggestion
Anyone can track the 2023 election process via [this GitHub issue](https://github.com/open-feature/community/issues/262).
We will ensure that all documents and assets related to the 2023 election process are public.
```

File: Elections/2023/Election-Guidelines.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+For the 2023 elections, [Helios Voting](https://vote.heliosvoting.org/) was chosen as it's a hosted solution with cryptographic guarantees that no GC members can meddle with the results.
+
+Helios voting also allows us to add GitHub handles to the list of voters in addition to email addresses. We need this, as we count contributions based on GitHub contributions and do not always have the contributor's actual email address. The disadvantage of Helios is that it does not support ranked voting
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:37]:
```suggestion
Helios voting also allows us to add GitHub handles to the list of voters in addition to email addresses.
We need this, as we count contributions based on GitHub contributions and do not always have the contributor's email address.
The disadvantage of Helios is that it does not support ranked voting.
```

File: Elections/2023/Election-Guidelines.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+# Nominations
+
+Anybody is eligible to run for the Governance Committee. During the "call for nomination" period, people can be nominated or nominate themselves by expressing their interest in [this issue](https://github.com/open-feature/community/issues/262) a Pull Request adding said candidate to the [candidates-2023.md](https://github.com/open-feature/community/Elections/2023/Candidates) file in the OpenFeature community repository. The template in that file includes the following columns:
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:41]:
```suggestion
Anybody is eligible to run for the Governance Committee. During the "call for nomination" period, people can be nominated or nominate themselves by expressing their interest in [this issue](https://github.com/open-feature/community/issues/262) a Pull Request adding said candidate to the [candidates-2023.md](https://github.com/open-feature/community/Elections/2023/Candidates) file in the OpenFeature community repository.

The template in that file includes the following columns:
```

File: Elections/2023/Election-Guidelines.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+The Pull Request will not be merged until the candidate has confirmed their desire to be nominated (if not self-nominating) and ratified via PR comments.
+
+# Voter Eligibility
+
+All [members of the OpenFeature Organization Members, Approvers, Maintainers will automatically be eligible to vote.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/e/OpenFeature2023) using their GitHub account. Voting will be approval voting, where each voter may select up to seven candidates, 4 from the current Governance Committee and three from the new nominees. The seven candidates (four from current GC and three from new nomineess) with the most votes win the election. The winners must be from different organizations, therefore only the highest ranked nominee from each organization will be elected. Nevertheless, it is possible to vote nominees from the same organization.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:57]:
```suggestion
Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/e/OpenFeature2023) using their GitHub account.
Voting will be approval voting, where each voter may select up to seven candidates, **four** from the current Governance Committee and three from the new nominees.
The seven candidates (four from current GC and three from new nominees) with the most votes win the election.
The winners must be from different organizations, therefore only the highest ranked nominee from each organization will be elected.
Nevertheless, it is possible to vote for nominees from the same organization.
```

File: Elections/2023/Election-Guidelines.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+To encourage diversity there will be a maximum of one-third representation on the Governance Committee from any one company at any time.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:58]:
```suggestion
To encourage diversity, there will be a maximum of one-third representation on the Governance Committee from any one company at any time. 
```

File: Elections/2023/Election-Guidelines.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+To encourage diversity there will be a maximum of one-third representation on the Governance Committee from any one company at any time. 
+If the outcomes of an election result in greater than 1/3 representation (or maximum of two, whichever is greater), the lowest vote getters from any particular company will be removed until representation on the committee is equal or less than one-third.
+
+If there is a draw between the lower ranked candidates, there will be a runoff election between the tied candidates. The runoff election will be held within 48 hours of the original election. The runoff election will be a simple majority vote, and the candidate with the most votes will be elected.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:61]:
```suggestion
If there is a draw between the lower ranked candidates, there will be a runoff election between the tied candidates.
The runoff election will be held within 48 hours of the original election.
The runoff election will be a simple majority vote, and the candidate with the most votes will be elected.
```

File: Elections/2023/Election-Guidelines.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+Per Helios, voting is entirely private: nobody will know any individual's vote.
+
+# Results
+
+Voting will close on the 17th December 2023 23:59 UTC. Nominees will be stack ranked. If a nominee becomes ineligible, the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:67]:
```suggestion
Voting will close on the 17th December 2023 23:59 UTC. Nominees will be stack ranked.
If a nominee becomes ineligible, the election committee will skip those nominees and pick the nominee with the next-highest score.
The exact scores for each candidate will be public.
```

File: Elections/2023/Election-Guidelines.md (hunk 2 above)
Comment (DavidPHirsch) [reply Elections/2023/Election-Guidelines.md]: was a typo

**Comment by DavidPHirsch** [comment]
@open-feature/governance-board

**Review by moredip — APPROVED** [review]

File: Elections/2023/Election-Guidelines.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
+# OpenFeature 2023 Governance Committee election
+
+Please see **[the list of candidates running in the election here](https://github.com/open-feature/community/pull/261/files#diff-aa2a6823369b7c294aa78e2867de50e1707b786d7b208ce5dd2269512765f598)**.
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:3]: This link will need to be updated once #261 has been merged

File: Elections/2023/Election-Guidelines.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+
+This election should fill seven seats: 
+- fours seats from the existing governance committee
```
Comment (beeme1mr) [inline Elections/2023/Election-Guidelines.md:6]:
```suggestion
- four seats from the existing governance committee
```

**Review by beeme1mr — APPROVED** [review]

File: Elections/2023/Election-Guidelines.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+# Voter Eligibility
+
+All members of the OpenFeature Organization Members, Approvers, Maintainers will automatically be eligible to vote.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/e/OpenFeature2023) using their GitHub account.
+Voting will be approval voting, where each voter may select up to seven candidates, **four** from the current Governance Committee and three from the new nominees.
+The seven candidates (four from current GC and three from new nominees) with the most votes win the election.
+The winners must be from different organizations, therefore only the highest ranked nominee from each organization will be elected.
```
Comment (justinabrahms) [inline Elections/2023/Election-Guidelines.md:65]: This contrasts with a few lines down where we say 2 from the same org is okay.

Comment (AloisReitbauer) [reply Elections/2023/Election-Guidelines.md:65]: I agree with Justin. Should we define, that for the same organisation only the candidate with the highest votes will be considered and multiple candidates only fi not all seats fill up?

**Review by AloisReitbauer — CHANGES_REQUESTED** [review]

File: Elections/2023/Election-Guidelines.md (hunk 12 above)
Comment (AlexsJones) [reply Elections/2023/Election-Guidelines.md:65]: +1

Comment (DavidPHirsch) [reply Elections/2023/Election-Guidelines.md]: updated

**Pull request merged**